### DISCUSSION TITLE
add string field length to 'basic' options for string fields

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13293] [Feature Requests]
Most times, this is the only reason I click on 'advanced' when setting up a field and is a really basic option. I think it'd be best if the string length was set as an option in the basic options with a default value of 255 preset.

**Comment by rijkvanzanten** [comment]
> and is a really basic option

While I personally agree, you might be surprised by how little people started caring about that level of optimizations on the database 🤔 

Lets let this simmer for a little bit to gauge community interest and we'll take it from there 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
